src/battery.c

```diff
@@ -0,0 +1,19 @@
+#include "battery.h"
+
+/**
+ * @brief ADC1 and ADC2 interruption routine.
+ *
+ * - Manage the ADC2 analog watchdog interruption flag.
+ * - Send a message.
+ * - Toggle a LED.
+ * - Disable analog watchdog interruptions on injected channels.
+ */
+void adc1_2_isr(void)
+{
+	if (adc_get_flag(ADC2, ADC_SR_AWD)) {
+		adc_clear_flag(ADC2, ADC_SR_AWD);
+		printf("battery_low\n");
```

Comment:
I'd rather print something like "WARNING: Battery low!\n". Later we could create some `log_warning()`, `log_error()` and `log_info` functions maybe?

Reply:
I created https://github.com/Theseus/theseus/issues/83.

Reply:
Changed, and good point to create log_error functions and levels.